docs/en/sensors-connectivity-setup.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Sensors Connectivity Module Setup
+
+contributors: [LoSk-p, Vourhey, tubleronchik]
+translated: true
+tools:   
+  - IPFS 0.8.0
+    https://docs.ipfs.io/install/command-line/
+  - sensors-connectivity 1.2.0
+    https://github.com/airalab/sensors-connectivity/tree/v1.2.0
+
+---
+
+Now let's start personal sensors Connectivity module.
+
+## Pre-requirements
+
+To build a python package the IPFS daemon has to be installed. **This module requires IPFS daemon version no greater than 0.8.x** . 
+So, in example, IPFS daemon version is 0.8.0. Assuming, you work on linux:
+
+```
+wget https://dist.ipfs.io/go-ipfs/v0.8.0/go-ipfs_v0.8.0_linux-amd64.tar.gz
+tar -xzf go-ipfs_v0.8.0_linux-amd64.tar.gz
+cd go-ipfs
+sudo bash install.sh 
+ipfs init
+```
+Also, we need to install `python3 dev` package and `pip`:
+
+```shell
+sudo apt install python3-dev
```

Comment:
Let's merge it into a single line command

Reply:
solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/58285cacfd4c7308b14a9952502745f80e082cc4